communication/mav_modes.h

```diff
@@ -222,13 +222,14 @@ typedef enum
 	CUST_DESCENT_TO_SMALL_ALTITUDE = 8,			///< First auto landing behavior
 	CUST_DESCENT_TO_GND = 16,					///< Second auto landing behavior, comes after DESCENT_TO_SMAL_ALTITUDE
 	
-	CUST_COLLISION_AVOIDANCE = 32,
-
-	CUST_BATTERY_LOW = 64,
-	CUST_FENCE_1 = 128,
-	CUST_HEARTBEAT_LOST = 256,
-	CUST_REMOTE_LOST = 512,
-	CUST_GPS_BAD = 1024
+	CUST_COLLISION_AVOIDANCE = 32,				///< Collision avoidance 
+
+	CUST_BATTERY_LOW = 64,						///< Battery low flag
+    CUST_FENCE_1 = 128,							///< Fence 1 violation flag
+    CUST_FENCE_2 = 256,							///< Fence 2 violation flag
+    CUST_HEARTBEAT_LOST = 512,					///< Heartbeat loss flag
+    CUST_REMOTE_LOST = 1024,					///< Remote lost flag
+    CUST_GPS_BAD = 2048							///< GPS loss flag
```

Comment:
Here I think it would be clearer if a `1` means something positive, and a `0` something negative.
So I would inverse CUST_HEARTBEAT_LOST, CUST_REMOTE_LOST, and CUST_GPS_BAD